src/main/java/br/com/moip/models/Account.java

```diff
@@ -0,0 +1,119 @@
+package br.com.moip.models;
+
+import br.com.moip.api.request.RequestMaker;
+import br.com.moip.api.request.RequestProperties;
+import br.com.moip.api.request.RequestPropertiesBuilder;
+import org.apache.http.entity.ContentType;
+
+import java.util.Map;
+
+public class Account {
+
+    private static final String ENDPOINT = "/v2/accounts";
+    private static final ContentType CONTENT_TYPE = ContentType.APPLICATION_JSON;
+    private RequestMaker requestMaker;
+
+    /**
+     * This method is used to validate the argument of bellow method, if probably it's a tax document or not.
+     *
+     * @param   argument
+     *          {@code String} the received argument.
+     *
+     * @return  {@code boolean}
+     */
+    private boolean isTaxDocument(String argument) {
+        try {
+            Integer.parseInt(argument.substring(0,1));
+        } catch (Exception ex) {
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * This method allows you to check if a person already has a Moip Account, by it's tax document or e-mail.
+     * The tax document must be write with punctuation, for example: 123.456.789-00.
+     *
+     * @param   argument
+     *          {@code String} the person's tax document or e-mail.
+     *
+     * @param   setup
+     *          {@code Setup} the setup object.
+     *
+     * @return  {@code Map<String, Object>}
+     */
+    public Map<String, Object> checkExistence(String argument, Setup setup) {
```

Comment:
This method can be refactored as we talked

Reply:
Done.